test/puppetlabs/trapperkeeper/services/status/status_proxy_service_test.clj

```diff
@@ -30,7 +30,9 @@
                         :host "0.0.0.0"}
    :status-proxy       {:proxy-target-url "https://0.0.0.0:9001/ssl-status"
                         :ssl-opts common-ssl-config}
-   :web-router-service {:puppetlabs.trapperkeeper.services.status.status-proxy-service/status-proxy-service "/status-proxy"}})
+   :web-router-service {:puppetlabs.trapperkeeper.services.status.status-proxy-service/status-proxy-service "/status-proxy"}
+   ; Tell TK to not log errors to the screen. For tests with intentionally broken services
```

Comment:
@jpinsonault I think the way we would normally mute these log messages is by wrapping the relevant portion of the test in a call to `with-test-logging`. This captures log messages from TK and allows you to write assertions that certain messages were logged, and it also has the effect of stopping those messages from being logged to the console.